src/interface.c

```diff
@@ -40,14 +41,16 @@ void em_init(emacs_env *env)
     em_libgit_object_p = GLOBREF(INTERN("libgit-object-p"));
     em_libgit_repository_p = GLOBREF(INTERN("libgit-repository-p"));
     em_libgit_reference_p = GLOBREF(INTERN("libgit-reference-p"));
-
+    em_signature_p = GLOBREF(INTERN("libgit-signature-p"));
```

Comment:
I don't think you need this. But if you do, at least you need to declare the variable.

Reply:
Declared the variable.

As far as needing it, I honestly only put it in because this is my first time working with code like this and I wanted to follow the instructions exactly.  However, I see that you're using the _p versions in the ASSERT_* macros.  I missed that the first time around.  I assume I should be doing the same thing?